distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Dialect.enso

```diff
@@ -1,23 +1,22 @@
 from Standard.Base import all hiding First, Last
 
 from Standard.Table.Data.Aggregate_Column.Aggregate_Column import all
-import Standard.Database.Data.SQL
-from Standard.Database.Data.SQL import SQL_Type, Statement, code
-import Standard.Database.Data.Dialect
-import Standard.Database.Internal.Base_Generator
-import Standard.Database.Internal.IR
-from Standard.Database.Errors import Unsupported_Database_Operation_Error_Data
+
+import project.Data.SQL_Statement.SQL_Statement
+import project.Data.SQL_Type.SQL_Type
+import project.Internal.Base_Generator
+import project.Internal.IR.Internal_Column.Internal_Column
+import project.Internal.IR.Order_Descriptor
+
+from project.Data.SQL import join, code
```

Comment:
I'm a bit hesitant about importing methods like these.
While ok with `code` as its used all over the place, I'd keep `join` qualified: `SQL.join` - as `join` itself doesn't tell clearly enough what it does (well, maybe we need to rename it to `join_sql_builders` or something like that then...). 
Also given that we don't have tooling support, seeing `SQL.join` makes it easier to find the definition of the method than seeing just `join`.